packages/asset-service/src/generateAssetData/ethTokens/yearnSdk.ts

```diff
@@ -0,0 +1,6 @@
+import { JsonRpcProvider } from '@ethersproject/providers'
+import { Yearn } from '@yfi/sdk'
+
+// YearnMarketCapService deps
+const provider = new JsonRpcProvider(process.env.REACT_APP_UNCHAINED_ETHEREUM_HTTP_URL)
+export const yearnSdk = new Yearn(1, { provider })
```

Comment:
can you comment or pull this magic number out to a const or something? i'm assuming this is mainnet at a first guess.

Reply:
Sure, done.